tests/testthat/test-ensembleMethods.R

```diff
@@ -176,22 +179,26 @@ context("Residual extraction")
 test_that("Residuals provided by residuals are proper for ensemble objects", {
   skip_on_cran()
   set.seed(2239)
-  ens.class <- caretEnsemble(models.class, trControl=trainControl(method="none"))
+  ens.class <- caretEnsemble(models.class, trControl = trainControl(method = "none"))
   models.subset <- models.reg[2:4]
   class(models.subset) <- "caretList"
-  ens.reg <- caretEnsemble(models.subset, trControl=trainControl(method="none"))
-  residTest <- residuals(ens.class)
-  residTest2 <- residuals(ens.reg)
+  ens.reg <- caretEnsemble(models.subset, trControl = trainControl(method = "none"))
+  suppressWarnings({
+    residTest <- residuals(ens.class)
+    residTest2 <- residuals(ens.reg)
+  })
   obs1 <- ifelse(Y.class == "No", 0, 1)
   obs2 <- Y.reg
-  predTest <- predict(ens.class)
-  predTest2 <- predict(ens.reg)
-  #expect_identical(residTest, obs1 - predTest)
-  #expect_identical(residTest2, obs2 - predTest2)
-  expect_false(identical(residTest2, predTest2 -obs2))
-
-  mr1 <- multiResiduals(ens.class)
-  mr2 <- multiResiduals(ens.reg)
+  expect_warning(predTest <- predict(ens.class))
+  expect_warning(predTest2 <- predict(ens.reg))
+  # expect_identical(residTest, obs1 - predTest)
+  # expect_identical(residTest2, obs2 - predTest2)
+  expect_false(identical(residTest2, predTest2 - obs2))
+
+  suppressWarnings({
```

Comment:
What warnings does this give now?

Reply:
If `multiResiduals` is called with only one argument, `predict` is called without `newdata` argument. Also, it is displayed an error from `lm` when calling `multiResiduals(ens.class)`.

Reply:
can we fix the warning or is it expected?

Reply:
added an issue to fix this warning in the future, as it comes up in a bunch of the tests and I (or someone) should fix it sometime: https://github.com/zachmayer/caretEnsemble/issues/262